NewForce-at-Mountwest/nutshell-team-hedgehopper-1
# Chat Functionality

You should see a blank chatbox and an input to enter text. 
-when text is entered chat should populate.
-users should only be able to edit their own messages.

to test:
- add many messages
- edit your messages
-try to edit others messages(should not work)
-try to break please
-test news article functionality
